src/microbe_stage/systems/FluidCurrentsSystem.cs

```diff
@@ -116,7 +180,19 @@ protected override void Update(float delta, in Entity entity)
         var pos = new Vector2(position.Position.X, position.Position.Z);
         var vel = VelocityAt(pos) * Constants.MAX_FORCE_APPLIED_BY_CURRENTS;
 
-        physicsControl.ImpulseToGive += new Vector3(vel.X, 0, vel.Y) * delta;
+        physicsControl.ImpulseToGive += new Vector3(vel.X, 0, vel.Y) * delta
+            * entity.Get<CurrentAffected>().EffectStrength;
```

Comment:
This would be the place to read effect strength into a variable and if it is 0 set it to 1 and if it is negative *then* skip applying any force (as -1 seems like a good value to indicate disabling as 0 can't be used for that).